.github/workflows/integ.yml

```diff
@@ -11,12 +11,29 @@ on:
       - ready_for_review
   merge_group: {}
 jobs:
+  targetenv:
+    permissions:
+      contents: read
+    outputs:
+      env_name: ${{ steps.undefined.outputs.undefined }}
```

Comment:
`undefined` ?

Reply:
Is projen trying to be helpful?

![image](https://user-images.githubusercontent.com/524162/235652657-e4cd316e-c979-4b6b-9da5-4eca63bb9ba1.png)

Reply:
It might be 🤔